lib/vdc/BOVCollection.cpp

```diff
@@ -218,6 +248,7 @@ int BOVCollection::_validateParsedValues()
     }
 
     // Validate endian type
+    if (_tmpDataEndian != _bigEndianString && _tmpDataEndian != _littleEndianString) return _invalidEndianError(ENDIAN_TOKEN);
```

Comment:
My opinion is to keep it, since we support endianness with our raw2vdc converter.  If we don't want to support this, we should deprecate it in the BOV reader and in raw2vdc.

Reply:
but there's no way to test if the code is correct or not. Actually, it has already caused a few bugs even before I could test if the correctness of big/small conversion code. So to reduce further bugs, deleting it is the best way.

Reply:
Endian stuff has been removed.

Reply:
Seems not totally removed?

Reply:
I believe you're looking at an outdated file.